components/gcp/dataproc/create_cluster/README.md

```diff
@@ -142,3 +138,13 @@ experiment = client.create_experiment(EXPERIMENT_NAME)
 run_name = pipeline_func.__name__ + ' run'
 run_result = client.run_pipeline(experiment.id, run_name, pipeline_filename, arguments)
 ```
+
+## References
+*   [Kubernetes Engine for Kubeflow](https://www.kubeflow.org/docs/started/getting-started-gke/#gcp-service-accounts)
```

Comment:
Not sure if the text and hyperlink co-relate?

Reply:
yes, we can remove this hyperlink, doesn't add any value